test/NuGet.Core.Tests/NuGet.ProjectModel.Test/LockFileParsingEnvironmentVariable.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Test.Utility;
+
+namespace NuGet.ProjectModel
+{
+    public class LockFileParsingEnvironmentVariable
```

Comment:
```suggestion
    public static class LockFileParsingEnvironmentVariable
```

All methods members are static, so the class can be declared as static too